modules/wallet/wallet.go

```diff
@@ -111,6 +111,7 @@ type Wallet struct {
 func (w *Wallet) Height() types.BlockHeight {
 	w.mu.Lock()
 	defer w.mu.Unlock()
+	w.syncDB()
```

Comment:
Why do we need to sync here?

Reply:
see above -- don't want to report information that won't survive a crash.